lib/src/server.dart

```diff
@@ -56,25 +56,54 @@ abstract class Service {
 /// Listens for incoming gRPC calls, dispatching them to a [ServerHandler].
 class Server {
   final Map<String, Service> _services = {};
-  ServerSocket _server;
+  final int port;
+  final SecurityContext _securityContext;
 
+  ServerSocket _insecureServer;
+  SecureServerSocket _secureServer;
```

Comment:
You'd think so, but no.

`ServerSocket` implements `Stream<Socket>`, and `SecureServerSocket` extends `Stream<SecureSocket>`, but `Stream` doesn't define a `close` method. They each define their own.